ner4opt/features.py

```diff
@@ -0,0 +1,592 @@
+import re
```

Comment:
just noticed one thing; should this file be featurizor.py per the Featurizer() class below?